source/extensions/quic_listeners/quiche/active_quic_listener.cc

```diff
@@ -112,7 +118,7 @@ void ActiveQuicListener::shutdownListener() {
 
 ActiveQuicListenerFactory::ActiveQuicListenerFactory(
     const envoy::config::listener::v3::QuicProtocolOptions& config, uint32_t concurrency)
-    : concurrency_(concurrency) {
+    : concurrency_(concurrency), enabled_(config.enabled()) {
```

Comment:
what if config.has_enabled() == false?

Reply:
Just verified the implementation, has_enabled() checks if field is not (either nullptr or is set to default value), but enabled instead always provides value to the caller:
```
inline const ::envoy::config::core::v3::RuntimeFeatureFlag& QuicProtocolOptions::_internal_enabled() const {
  const ::envoy::config::core::v3::RuntimeFeatureFlag* p = enabled_;
  return p != nullptr ? *p : *reinterpret_cast<const ::envoy::config::core::v3::RuntimeFeatureFlag*>(
      &::envoy::config::core::v3::_RuntimeFeatureFlag_default_instance_);
}
```
If config is empty, enabled field is set to default instance.